project/package/CodeEditor.vue

```diff
@@ -386,6 +390,21 @@ export default {
   margin: 0;
 }
 
+count-lines{
+  counter-reset: line;
+}
+.count-lines:deep(span.hljs-tag){
+  counter-increment: line;
+}
+.count-lines:deep(span.hljs-tag:before){
+  content: counter(line);
+  position: absolute;
+  left: 5px;
```

Comment:
For now not as important but perhaps make this into a new prop in the future so it's configurable.